core/tests/test_storage.cache.py

```diff
@@ -82,14 +82,11 @@ def call_Initialize(**kwargs):
             msg = Initialize(**kwargs)
             return await_result(handle_Initialize(DUMMY_CONTEXT, msg))
 
-        def call_ClearSession():
-            return await_result(handle_ClearSession(DUMMY_CONTEXT, ClearSession()))
-
         session_id = call_Initialize().session_id
         cache.set(KEY, "hello")
         self.assertEqual(cache.get(KEY), "hello")
 
-        call_ClearSession()
+        call_Initialize()
```

Comment:
please drop the whole test case, it's not doing anything other than the `test_Initialize` above it